README.md

```diff
@@ -0,0 +1,28 @@
+# Static App Template
+
+A template for building static apps with Gatsby.
+
+## Developing
+
+### Requirements
+
+- Docker
+- Docker Compose
```

Comment:
Note that Docker is used only for local development, not for running on the server (static deployments are handled with Netlify, where we can leverage their prebuilt Node environment). The advantage of a containerized setup is that we can ensure that dependencies are consistent across environments (e.g. make sure we all have the same Node.js version, and make sure that OS-specific NPM packages install properly) and we can reduce setup instructions to one step (`docker-compose build`).